src/components/EnhancedTable/EnhancedTable.md

```diff
@@ -3,21 +3,21 @@ Example:
     const Immutable = require('immutable');
 
     const columns = Immutable.fromJS([{
-      title: 'Name',
+      label: 'Name',
```

Comment:
To be honest this was kind of a stupid oversight. I'm probably going to change it back. It originally existed because of the _first_ table I made, which required `label` as a prop for a `Column` component, which was used specifically for column header rows.